src/styles/respec.css.js

```diff
@@ -222,15 +222,19 @@ aside.example .marker > a.self-link {
   border: none;
   color: inherit;
   position: relative;
-  top: -2.9em;
+  top: -2.75em;
   left: -1.4em;
   margin-bottom: -2em;
+  display: block;
```

Comment:
This should be an inline-block I think. It makes entire heading (even the whitespace after the heading) a link clickable area (and makes text selection difficult).